packages/browser-sdk/src/providers/embedded/index.ts

```diff
@@ -26,6 +27,13 @@ export class EmbeddedProvider extends CoreEmbeddedProvider implements Provider {
       urlParamsAccessor,
       stamper,
       name: platformName, // Use detected browser name and version for identification
+      analyticsHeaders: {
+        [ANALYTICS_HEADERS.SDK_TYPE]: "browser-sdk",
+        [ANALYTICS_HEADERS.PLATFORM]: platformName,
```

Comment:
is this the browser agent? should be more generic or it's gonna be hard to identify in sigma